### PR TITLE
feat(AttentionBoxLink): use link component

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/5835997259

### CODE REVIEW

File: src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.module.scss [hunk 1]
```diff
@@ -1,4 +1,7 @@
-a.attentionBoxLink {
+// TODO: use just a class (without [href]) after removing anchor style from Typography
+// Using [href] to increase specificity and override `Typography a` styles
+a[href].attentionBoxLink,
+a[href].attentionBoxLink:hover {
```
Comment (talkor) [inline src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.module.scss:4]: Anchor style from `Typography` is overriding it so I had to make it more specific, also added a comment to remove anchor style from Typography as users can just use the Link component

File: src/components/AttentionBox/__stories__/attentionBox.stories.js [hunk 2]
```diff
@@ -96,25 +98,17 @@ export const States = {
...
 
 export const AttentionBoxWithLink = {
-  /**Class name for setting a constant width to the attention box*/
   render: () => {
     return (
-      <AttentionBox compact icon={Info} className="monday-storybook-attention-box--fixed-width">
-        <Flex
-          style={{
-            width: "100%"
-          }}
```
Comment (talkor) [inline src/components/AttentionBox/__stories__/attentionBox.stories.js:106]: No effect on story so I removed

File: src/components/AttentionBox/__stories__/attentionBox.stories.js [hunk 3]
```diff
@@ -96,25 +98,17 @@ export const States = {
 };
 
 export const AttentionBoxWithLink = {
-  /**Class name for setting a constant width to the attention box*/
   render: () => {
     return (
-      <AttentionBox compact icon={Info} className="monday-storybook-attention-box--fixed-width">
```
Comment (talkor) [inline src/components/AttentionBox/__stories__/attentionBox.stories.js:102]: Icon isn't shown in this variation so I removed

File: src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx
Comment (YossiSaadi) [inline src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx:1]: WELL DONE

File: src/components/AttentionBox/__stories__/attentionBox.stories.js [hunk 4]
```diff
@@ -96,25 +96,17 @@ export const States = {
...
-        <Flex
-          style={{
-            width: "100%"
-          }}
-          justify={Flex.justify.SPACE_BETWEEN}
-          gap={Flex.gaps.XS}
-        >
+      /** Classname for setting a constant width to the attention box */
```
Comment (YossiSaadi) [inline src/components/AttentionBox/__stories__/attentionBox.stories.js:101]: Notice you removed the icon from this story, is this intentional?

Comment (YossiSaadi) [reply src/components/AttentionBox/__stories__/attentionBox.stories.js:101]: oh I saw the other comment nm

**Review by YossiSaadi — APPROVED** [review]

File: src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
-}
-const AttentionBoxLink: FC<AttentionBoxLinkProps> = ({ href, text }) => {
+const AttentionBoxLink: FC<LinkProps> = ({
+  href,
+  text,
+  // TODO: use Link's target default in next major
+  // For backward compatibility - using _self as default
+  target = Link.target.SELF,
+  className,
+  ...linkProps
```
Comment (YossiSaadi) [inline src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx:13]: make sure nothing is required btw
as it would be breaking

Comment (talkor) [reply src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx:13]: Yes thanks I checked, all Link props are optional

**Pull request merged**